### PR TITLE
Seeds.rb, Added users and models for departments

### PR DESCRIPTION
**Pull request by kgf**
to seeds.rb Not finished yet, but still helpful

### CODE REVIEW

**Comment by kgf** [comment]
Anyone should feel free to review, but I know Chris was particularly interested.

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,85 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# I have added some objects whose status is Inactive to ensure they don't appear.
+# Need to create
```
Comment (houhoulis) [inline db/seeds.rb]: I think you created these things.  So I guess this comment is outdated?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,85 @@
...
+end
+
+Role.find_or_create_by(name: "Editor")
+Role.find_or_create_by(name: "Author")
+Role.find_or_create_by(name: "Depositor")
+Role.find_or_create_by(name: "Trainer")
+Role.find_or_create_by(name: "Admin")
+
+# Departments are where people call home.
+# I need another status to determine visbility on the various drop downs.
```
Comment (houhoulis) [inline db/seeds.rb]: "visibility" (spelling)

But also, I don't understand this comment.  Is this a TODO?

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,85 @@
...
+Department.create([
+  {name: "Community Emergency Response Team", status: "Active"},
+  {name: "Medical Reserve Corp", status: "Active"},
+  {name: "Department Public Works", status: "Active"},
+  {name: "Inactive Department", status: "Inactive"}
+  ])
+
+# There does seem to be the correct link between skills and courses
+# As I create a person, I wonder what
```
Comment (houhoulis) [inline db/seeds.rb]: Many of the comments in here seem like notes to yourself.  At least, I am not sure what they mean.  They could either be fleshed out, addressed, or removed.

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,85 @@
...
+Role.find_or_create_by(name: "Admin")
+
+# Departments are where people call home.
+# I need another status to determine visbility on the various drop downs.
+Department.create([
+  {name: "Community Emergency Response Team", status: "Active"},
+  {name: "Medical Reserve Corp", status: "Active"},
+  {name: "Department Public Works", status: "Active"},
+  {name: "Inactive Department", status: "Inactive"}
+  ])
```
Comment (houhoulis) [inline db/seeds.rb:37]: Cool, I had no idea you could do

``` ruby
Foo.create([
  foo_attributes,
  foo_2_attributes,
  foo_3_attributes...
])
```

👍

**Pull request merged**